src/java.desktop/share/classes/java/beans/beancontext/BeanContextServicesSupport.java

```diff
@@ -148,7 +147,7 @@ public BeanContextServices getBeanContextServicesPeer() {
 
     /************************************************************************/
 
-    /*
+    /**
```

Comment:
This will add many statements to the specification which could be verified by the TCK. DO we really want to say all of that? Especially about serialization. Same question for other changes below.

Reply:
ok. only added relevant info